CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 * #1012 Return status for introspecting a nonexistent token from 401 to the correct value of 200 per [RFC 7662](https://datatracker.ietf.org/doc/html/rfc7662#section-2.2).
-* #1068 Revert #967 which incorrectly changed an API. See #1066.
+* #1069 Re-introduce support for traditional get_additional_claims API. See #1066 for what happened between 1.6.0 and now.
```

Comment:
Since this got reverted in 1.6.2 there's no need to document this as a fix here.